packages/core/src/api/exporters/copyExtension.ts

```diff
@@ -7,9 +7,12 @@ import type { BlockNoteEditor } from "../../editor/BlockNoteEditor";
 import { BlockSchema, InlineContentSchema, StyleSchema } from "../../schema";
 import { initializeESMDependencies } from "../../util/esmDependencies";
 import { createExternalHTMLExporter } from "./html/externalHTMLExporter";
-import { createInternalHTMLSerializer } from "./html/internalHTMLSerializer";
 import { cleanHTMLToMarkdown } from "./markdown/markdownExporter";
 
+// use a dynamic import because we want to access
+// __serializeForClipboard which is not exposed in types
+let pmView: any;
```

Comment:
instead of a dynamic import (which also changes the way the module is loaded), I think you can do

```
import * as pmView from "prosemirror-view;
(pmView as any).__serializeForClipboard
```